async-inference/Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Use current region rather than hard-coding us-west-2.</p><pre class="ql-syntax" spellcheck="false">boto_session = boto3.session.Session()
region = boto_session.region_name
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
ACK